railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
@fxn What I'm saying here is people might have a config for `BUNDLE_JOBS` which is higher than the number of cpus. It's safe to set to 2 or 4 or even higher even if there's only one cpu available (e.g. development on vagrant).

I usually set `BUNDLE_JOBS` to the result of `number of cpus * 8` and don't want rails to ignore it. At least it should respect the existing config.